### DISCUSSION TITLE
Make sure the cloned card is writeable after cloning

### DISCUSSION THREAD

**Discussion by sesardelaisla** [ikarus23/MifareClassicTool#470] [Q&A]
Hello Everyone!

Just discovered this tool and it looks great! My purpose is to be able to clone cards. For the moment, I have been able to read a Mifare wristband I would like to clone to another wristband (unit 0 programmable). However, I haven't tried to write it yet. I would like to make sure that the destination wristband is not set as read only. The wristband I would like to clone from is read only. I see some info about this topic in the documentation, although it is confusing for me. How can I make sure that destination can be written again or formatted back? Thanks!

**Comment by ikarus23** [comment]
Hi. Sorry for the late response.

You can achieve what you are trying to to with a special option. Under "Write Tag" -> "Write Dump (Clone)" -> "Show options" you will find a "Use Access Conditions for all sectors"-option. Permissions - like read only - are defined by the Access Conditions. This option allows you to use the defined AC instead of the ones that are in the dump file (original wristband). 

The field is prefilled with `08778F`. This is a pretty common configuration. It allows you to change everything with key B. So it is not read only. You can decode the cryptic ACs with the "Access Condition De-/Encoder" under "Tools". Or you can use online tools like https://slebe.dev/mifarecalc/.

Hope this helps!
